jenkins_cli/cli.py

```diff
@@ -31,6 +32,11 @@
                   }
 
 
+# Default constants to parse booleans from .jenkins-cli
+TRUE = ['yes', 'Yes', 'YES', 'True', 'true', 'TRUE', '1']
```

Comment:
Well, I think it will be very weird if someone configures this property like this: `ignore_ssl=False` and the application continue ignoring SSL errors.

That's why I thought it would be a good idea to configure a list with all acceptable strings.

`ignore_ssl=` also seems strange to me. What do you think?